auth_proxy/proxy/flask.py

```diff
@@ -99,3 +105,32 @@ def _patient_security(self):
             return 'Patient/{}'.format(patient)
         except AttributeError:
             return 'public'
+
+
+class OpenFlaskClient(FlaskClient):
```

Comment:
Nitpick, but since we're doing something as drastic as completely removing all permission checks on health data, I think it makes sense to call this class something more alarming, like `UnsecureFlaskClient`. "Open" just doesn't feel dangerous enough.

Reply:
Good point. Updated!